crates/ink/codegen/src/generator/trait_def/definition.rs

```diff
@@ -38,6 +38,7 @@ impl<'a> TraitDefinition<'a> {
             format_ident!("{}Output", ident.to_string().to_lower_camel_case());
         quote_spanned!(span =>
             /// Output type of the respective trait message.
+            #(#attrs)*
```

Comment:
Maybe you want to use `cfg_attrs` here because it is generated part of the trait(not expected by the user)?